content/_data/events/2019-04-16-denver-jam.adoc

```diff
@@ -0,0 +1,6 @@
+---
+title: "Jenkins X Quiz Show!"
```

Comment:
Is this what you were recommending @oleg-nenashev ?  If so, this would be great with me.
```suggestion
title: "Denver JAM"
```